src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -49,6 +49,7 @@ interface IBlazor {
     dotNetCriticalError?: any
     getSatelliteAssemblies?: any,
     sendJSDataStream?: (data: any, streamId: string, chunkSize: number) => void,
+    sendJSDataStreamWebview?: (data: any, streamId: string, chunkSize: number, jsDataStream: any) => void,
```

Comment:
This is so we can provide a `JSDataStream` `DotNetObjectReference` to call `ReceiveJSDataChunk` on.